### ISSUE TITLE
9.0.0 version of Windows PV drivers

### ISSUE THREAD

**Issue by marmarek** [QubesOS/qubes-issues#7494]
> This is already fixed upstream in 9.0.0 version of PV drivers

We tried release 9.0.0, but there were significant problems
- inability to boot after uninstalling them
- max-grant-frames value must be less than 511
 
This may have already been fixed in the master, but we need the signed ones to use. I'll try to test the latest master builds.

_Originally posted by @jevank in https://github.com/QubesOS/qubes-issues/issues/7404#issuecomment-1117785474_

**Comment by marmarek** [comment]
Lets track PV drivers update here, even if not actionable right now.

**Comment by DemiMarie** [comment]
> This may have already been fixed in the master, but we need the signed ones to use. I'll try to test the latest master builds.

Is disabling signature checking an option?

**Comment by jevank** [comment]
> Is disabling signature checking an option?

Yes, this is an option, we have a tool to make it on the fly ([pkihelper](https://github.com/QubesOS/qubes-gui-agent-windows/tree/master/install-helper/pkihelper)). But I was thinking Qubes OS compatibility could be the reason to make release after almost 3 years :)

**Comment by DemiMarie** [comment]
> > Is disabling signature checking an option?
> 
> Yes, this is an option, we have a tool to make it on the fly ([pkihelper](https://github.com/QubesOS/qubes-gui-agent-windows/tree/master/install-helper/pkihelper)). But it requires system preparation with reboot and I was thinking Qubes OS compatibility could be the reason to make release after almost 3 years :)

Is this sufficient for kernel-mode code signing?

**Comment by jevank** [comment]
> Is this sufficient for kernel-mode code signing?

With [test signing mode ](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/the-testsigning-boot-configuration-option) enabled only.

**Comment by ben-grande** [comment]
@jevank Please retest.
@omeg Do you think it is solved.

If this is solved, please close.

**Comment by marmarek** [comment]
While new QWT uses (much) newer PV drivers, there is also https://github.com/QubesOS/qubes-vmm-xen-windows-pvdrivers/pull/4 which is not merged due to some issues it introduces (see comments there).
